code-challenges.test.js

```diff
@@ -75,4 +75,4 @@ function votesRemaining(votesTotal) {
 //1.created a function votesRemaining that takes in the paramiter votesToal
 //2.created another variable to descontructing both of them so theyre a single 
 //3.another var remaining to = upVotes - downVotes and return the total
-//4.return updated array 
\ No newline at end of file
+//4.return updated array 
```

Comment:
Hey, nice work.  Be sure to run your tests - your code in question 1 didn't pass.  Which is ok, if you didn't get it, but please explain your troubleshooting process or thoughts on it.  Also, you can console log  every line in your code to see where the error is.  Give a shot.  :). Way to get the bottom one.  I'd encourage you to refactor to up your skill level.